processor/attributekeyprocessor/attributekeyprocessor.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package attributekeyprocessor
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/census-instrumentation/opencensus-service/consumer"
+	"github.com/census-instrumentation/opencensus-service/data"
+	"github.com/census-instrumentation/opencensus-service/processor"
+)
+
+// KeyReplacement identifies a key and its respective replacement.
+type KeyReplacement struct {
+	// Key the attribute key to be replaced.
+	Key string `mapstructure:"key"`
+	// NewKey is the value that will be used as the new key for the attribute.
+	NewKey string `mapstructure:"replacement"`
```

Comment:
Perhaps for consistency name this field `Replacement` instead of `NewKey`.

Reply:
Internally I prefer `NewKey` to emphasize that it is a new entry the map, for the config itself `new-key` is a implementation detail.